src/frequenz/sdk/timeseries/_formula_engine/_formula_engine.py

```diff
@@ -374,6 +374,24 @@ def new_receiver(
         recv._engine_reference = self  # type: ignore # pylint: disable=protected-access
         return recv
 
+    @staticmethod
+    def from_receiver(receiver: Receiver, name: str) -> FormulaEngine:
+        """
+        Create a formula engine from a receiver.
+
+        Can be used to compose a formula engine with a receiver.
+
+        Args:
+            receiver: A receiver that streams `Sample`s.
+            name: A name for the formula engine.
+
+        Returns:
+            A formula engine that streams the `Sample`s from the receiver.
+        """
+        builder = FormulaBuilder(name)
+        builder.push_metric(name, receiver, nones_are_zeros=False)
```

Comment:
Shouldn't `nones_are_zeros` be an option to this method?

Reply:
You are right!